packages/swapper/src/swappers/ChainflipSwapper/utils/getQuoteOrRate.ts

```diff
@@ -297,6 +297,15 @@ export const getQuoteOrRate = async (
         singleQuoteResponse.boostQuote.egressAmountNative!,
       )
 
+      // This is not really a buyAmount before fees but rather an input/output calculation to get the sell amount
+      // prorated to the buy asset price to determine price impact
```

Comment:
Also another thing to be mindful of here is that our market price and Chainflip's will always diverge - if we're trying to get a reliable input/output, we should probably use our own, otherwise we could and will end with wrong amounts (in the worst case scenario, the delta we show for percentage could even be the abs() version of the real one e.g negative instead of positive impact).

Can we dependency-inject *our* buy/sell fiat ratio as something like `buyAssetSellAssetFiatRate` or similar?